backend/EasyMeets.Core/EasyMeets.Core.BLL/MappingProfiles/AvailabilityProfile.cs

```diff
@@ -13,7 +13,7 @@ public AvailabilityProfile()
             CreateMap<AvailabilitySlot, AvailabilitySlotDto>()
                 .ForMember(dest => dest.Members, opt => opt.MapFrom(src => src.SlotMembers.Select(x => x.User)))
                 .ForMember(dest => dest.EmailTemplateSettings, opt => opt.MapFrom(src => src.EmailTemplates))
-                .ForMember(dest => dest.Schedule, opt => opt.MapFrom(s => s.SlotMembers.First().Schedule));
+                .ForMember(dest => dest.Schedule, opt => opt.MapFrom(s => s.SlotMembers.FirstOrDefault()!.Schedule));
             CreateMap<AvailabilitySlotDto, AvailabilitySlot>();
```

Comment:
Wouldn't it still throw an error because of ! operator?

Reply:
I tested and didn't get any errors

Reply:
Ok

Reply:
You cannot use ! with FirstOrDefault